### PR TITLE
{chem} [intel/2015a] CP2k-20150904 with PLUMED

### PR DESCRIPTION
**Pull request by wpoely86**
Needs hpcugent/easybuild-easyblocks#681

### CODE REVIEW

File: easybuild/easyconfigs/c/CP2K/CP2K-20150904-intel-2015a-PLUMED.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+]
+
+builddependencies = [
+    ('flex', '2.5.39'),
+    ('Bison', '3.0.2'),
+]
+
+# don't use parallel make, results in compilation failure
+# because Fortran module files aren't created before they are needed
+parallel = 1
```
Comment (boegel) [inline easybuild/easyconfigs/c/CP2K/CP2K-20150904-intel-2015a-PLUMED.eb]: I don't think this is needed anymore, should work without it (I was told a while ago that parallel building works fine for recent CP2K versions)

so, let's chuck it and see what happens

File: easybuild/easyconfigs/c/CP2K/CP2K-20150904-intel-2015a-PLUMED.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+name = 'CP2K'
+version = '20150904'
+versionsuffix = '-PLUMED'
```
Comment (boegel) [inline easybuild/easyconfigs/c/CP2K/CP2K-20150904-intel-2015a-PLUMED.eb]: include version number (do it the way we do with Python?)

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in this PR)
Linux debian 8.1, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.9
See https://gist.github.com/84ebd6c69afc952d9e30 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in this PR)
Linux debian 8.1, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.9
See https://gist.github.com/400df0f8fa8325a9cc72 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in this PR)
Linux debian 8.1, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.9
See https://gist.github.com/108408a72f2360079ab1 for a full test report.

**Comment by boegel** [comment]
@wpoely86: awaiting a successful test report before I reconsider this

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in this PR)
Linux debian 8.1, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.9
See https://gist.github.com/e64aaaf7d6c04655bb06 for a full test report.

**Comment by wpoely86** [comment]
@boegel It's ready but the sandbox script failed when submitting the test report for some weird reason (timeout). Gonna try it again but the build works.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
Linux debian 8.1, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.9
See https://gist.github.com/d2c044f941c9c1227c75 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7a75922133999ce03605 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/0ebdbf5c58e7821c98ae for a full test report.

File: easybuild/easyconfigs/c/CP2K/CP2K-20150904-intel-2015a-PLUMED-2.1.3.eb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+# we use the git mirror: this is r15816 or 68331ef35b7
+sources = ['68331ef35b7.zip']
+source_urls = ['https://github.com/cp2k/cp2k/archive/']
+
+patches = [
+    'CP2K-%(version)s-ifort-compiler-bug-fix.patch',
+    'CP2K-2.4.0-fix_compile_date_lastsvn.patch',
+]
+
+unpack_options = ' && mv cp2k-68331* cp2k'
```
Comment (boegel) [inline easybuild/easyconfigs/c/CP2K/CP2K-20150904-intel-2015a-PLUMED-2.1.3.eb]: add a clarifying comment please, mention the error you're dancing around here (`Argument list too long`?)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b18b6547ba584f94445b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/9c377a8edffcfd7cb81c for a full test report.

**Comment by boegel** [comment]
Going in now https://github.com/hpcugent/easybuild-easyblocks/issues/681 is merged, thanks a bunch @wpoely86!

**Pull request merged**